lib/transformer/caster/to_atlas_attribute.rb

```diff
@@ -1,34 +1,29 @@
 module Transformer
   module Caster
-    class ToAtlasAttribute < Base
-      def analyze
-        static.merge(area_attributes).merge(init: initializer_inputs)
+    class ToAtlasAttribute
+      def self.filter(dataset_cast, template)
+        new(dataset_cast, template).filter
```

Comment:
@antw This is a class where I also have existential doubts over 😆 . Does this need to co-exist with the template. Or should I move all the filtering in with the template?